connector/github/github.go

```diff
@@ -372,6 +376,81 @@ func (c *githubConnector) groupsForOrgs(ctx context.Context, client *http.Client
 	return groups, fmt.Errorf("github: user %q not in required orgs or teams", userName)
 }
 
+func (c *githubConnector) userGroups(ctx context.Context, client *http.Client) (groups []string, err error) {
+	orgs, err := c.userOrgs(ctx, client)
+	if err != nil {
+		return groups, err
```

Comment:
[nit] How about `return nil, err`? A lot less _"huh, what's `groups`?"_ when reading this 😉

Reply:
Done